aten/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ else()
   cmake_dependent_option(
       NO_MKLDNN "Legacy no MKLDNN" OFF
       "USE_MKLDNN" ON)
+  add_subdirectory(../c10 c10)
```

Comment:
Seems weird to me to include stuff from parent directory. Shouldn't the parent directory do `add_subdirectory(c10)` instead?